tests/scripts/all.sh

```diff
@@ -3741,20 +3741,12 @@ support_test_psa_compliance () {
     [ "$ver_major" -eq 3 ] && [ "$ver_minor" -ge 10 ]
 }
 
-component_test_corrected_code_style () {
-    ./scripts/code_style.py --fix
-
-    msg "build: make, default config (out-of-box), corrected code style"
-    make
-
-    msg "test: main suites make, default config (out-of-box), corrected code style"
-    make test
-
-    # Clean up code-style corrections
-    git checkout -- .
+component_check_code_style () {
```

Comment:
Not sure why GitHub thinks this is part of the PR - probably not rebased off latest development?

Anyway, these changes are already in `development`, so should merge cleanly